packages/communication-react/review/communication-react.api.md

```diff
@@ -1439,6 +1440,15 @@ export interface MeetingCompositeProps extends BaseCompositeProps<CallCompositeI
     meetingInvitationURL?: string;
 }
 
+// @public
+export interface MeetingCompositeStrings {
+    chatButtonLabel: string;
+    chatPaneTitle: string;
+    peopleButtonLabel: string;
+    peoplePaneSubHeader: string;
```

Comment:
nit: 
Title vs SubHeader. Should this be SubTitle?

Reply:
+1 to SubTitle